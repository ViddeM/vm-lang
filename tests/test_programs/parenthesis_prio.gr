fn main() {
    print_number(2 * 5 + 1);
    print_number(2 * (5 + 1));
    print_number(1 + 2 * 5 + 1);
    print_number((1 + 2) * 5 + 1);
    print_number(1 + 2 * (5 + 1));
    print_number((1 + 2) * (5 + 1));
}
