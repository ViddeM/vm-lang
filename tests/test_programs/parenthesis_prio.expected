11
12
12
16
13
18